test/services/journeyAnswers/IncomeAnswersServiceImplSpec.scala

```diff
@@ -48,6 +55,26 @@ import scala.concurrent.Future
 class IncomeAnswersServiceImplSpec extends AnyWordSpecLike with Matchers with MacroBasedMatchers {
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
+  "getPrepopAnswers" should {
+    val filledIncomes: IncomesType = IncomesType(Some(100), Some(50))
+    "return empty answers if there is no answers submitted" in new TestCase() {
+      service.getPrepopAnswers(journeyCtxWithNino).value.futureValue shouldBe IncomePrepopAnswers(None, None).asRight
+    }
+
+    "return error if error is returned from the Connector" in new TestCase(connector = StubSelfEmploymentConnector(getPeriodicSummaryDetailResult =
+      Future(SingleDownstreamError(INTERNAL_SERVER_ERROR, SingleDownstreamErrorBody.parsingError).asLeft))) {
```

Comment:
Could be worth pulling out this error to a simple val, seeming as it is used a lot in the tests.